wrappers/Julia/CoolProp.jl

```diff
@@ -297,7 +297,7 @@ end
 # rhomolar The pointer to the array of molar density
 # hmolar The pointer to the array of molar enthalpy
 # smolar The pointer to the array of molar entropy
-function AbstractState_update_and_common_out(handle::Clong, input_pair::Clong, value1::Array{Real}, value2::Array{Real}, length::Real, T::Array{Real}, p::Array{Real}, rhomolar::Array{Real}, hmolar::Array{Real}, smolar::Array{Real})
+function AbstractState_update_and_common_out{R<:Real}(handle::Clong, input_pair::Clong, value1::Array{R}, value2::Array{R}, length::Real, T::Array{R}, p::Array{R}, rhomolar::Array{R}, hmolar::Array{R}, smolar::Array{R})
```

Comment:
http://docs.julialang.org/en/release-0.4/manual/methods/#parametric-methods

Float64 <: Real => true
Array{Float64 } <: Array{Real} => false
Because of Julia's invariance